server/store.go

```diff
@@ -65,7 +65,13 @@ func (p *Plugin) fetchOAuthUserInfo(tokenKey, userID string) (*zoom.OAuthUserInf
 }
 
 func (p *Plugin) disconnectOAuthUser(userID string) error {
+	// according to the definition encoded would be nil
+	// that means that the account is not linked
 	encoded, err := p.API.KVGet(zoomUserByMMID + userID)
+	if len(encoded) == 0 {
+		return errors.Wrap(err, "Your Zoom account is not currently linked")
```

Comment:
This will panic if `err` == `nil`. Why does the code `Wrap` the error?

Also: Do error checking first?

Reply:
actually this makes sense. I read the docs for KVGet and in this case `encoded` should be `nil`. Will handle the errors appropriately